hera_corr_cm/hera_corr_cm.py

```diff
@@ -320,77 +345,115 @@ def get_ant_status(self):
+            'pam_id': ('pam{$PF}_id', json.loads),
+            'fem_temp': ('fem{$PF}_temp', float),
+            'fem_voltage': ('fem{$PF}_voltage', float),
+            'fem_current': ('fem{$PF}_current', float),
+            'fem_pressure': ('fem{$PF}_pressure', float),
+            'fem_humidity': ('fem{$PF}_humidity', float),
+            'fem_id': ('fem{$PF}_id', json.loads),
+            'fem_switch': ('fem{$PF}_switch', str),
+            'fem_imu_theta': ('fem{$PF}_imu_theta', float),
+            'fem_imu_phi': ('fem{$PF}_imu_phi', float),
+            'timestamp': ('timestamp', dateutil.parser.parse),
+            'clip_count': ('eq_clip_count', int),
+            'fft_of': ('fft_overflow', lambda x: (x == 'True'))
         }
-        rv = {}
-        for host, val in stats.items():
-            rv[host] = {}
-            for key, convfunc in conv_methods.items():
-                try:
-                    rv[host][key] = convfunc(stats[host][key])
-                except:
-                    rv[host][key] = "None"
-        return rv
-
-    def get_snaprf_status(self):
+        ant_status = {}
+        for ant, vals in ant_to_snap.items():
+            for pol, hostinfo in vals.items():
+                antpol = "{}:{}".format(ant, pol)
+                host = hostinfo['host']
+                if host not in stats:
+                    continue
+                stream = hostinfo['channel']
+                antid = stream // 2
+                ant_status[antpol] = {'f_host': host, 'host_ant_id': stream}
+                for key, (ckey, cfunc) in conv_info.items():
+                    ckey = ckey.replace('{$CH}', str(stream))
+                    ckey = ckey.replace('{$PF}', str(antid))
+                    ckey = ckey.replace('{$POL}', pol)
+                    try:
+                        ant_status[antpol][key] = cfunc(stats[host][ckey])
+                    except:
+                        ant_status[antpol][key] = 'None'
+        return ant_status
+
+    def get_snaprf_status(self, numch=6):
         """
         Return a dictionary of SNAP input stats.
 
         Keys of returned dictionaries are of the form "<SNAP hostname>:"<SNAP input number>".
         Values of this dictionary are status key/val pairs.
 
         These keys are:
-            eq_coeffs (list of floats) : Digital EQ coefficients for this antenna
-            histogram (list of ints) : Two-dim list: [[bin_centers][counts]] represent ADC histogram
+            eq_coeffs (list of floats) : Digital EQ coefficients for this host
+            histogram (list of ints) : List of counts representing ADC histogram
             autocorrelation (list of floats) : Autocorrelation spectrum
             timestamp (datetime) : Asynchronous timestamp that these status entries were gathered
-
+            mean (float): mean of power for this host:stream
+            rms (float): rms of power for this host:stream
+            power (float): total power for this host:stream
```

Comment:
If these describe the dict returned by this function, I'd put them under the Returns section.